src/testHelpers/src/main/kotlin/org/vaccineimpact/orderlyweb/test_helpers/http/HttpClient.kt

```diff
@@ -29,7 +25,9 @@ object HttpClient
             .url(url)
             .headers(headers.toHeaders())
             .build()
-        return Response(client.newCall(request).execute())
+        client.newCall(request).execute().use {
```

Comment:
The changes here are not related to the main code change, but we were still getting loads of "leaked connection" warnngs in the tests. this was because the `Response` class here needed to be closed in order to close the underlying `OkHttpResponse`, but this was basically just not happening almost anywhere. I refactored this so that all the response closing logic is contained hgere and individual tests don't have to worry about it (since they weren't anyway).